system-addon/content-src/components/Base/_Base.scss

```diff
@@ -46,3 +46,11 @@ main {
     vertical-align: middle;
   }
 }
+
+.body-wrapper {
+  opacity: 0;
```

Comment:
Oh. ha ha, so the prerendered content can still trigger scrollbars without shifting ;)